istio/discovery.go

```diff
@@ -83,6 +92,14 @@ func (in *Discovery) getControlPlaneConfiguration(kubeCache cache.KubeCache, con
 		return nil, err
 	}
 
+	certConfigMap, err := kubeCache.GetConfigMap(controlPlane.IstiodNamespace, certificatesConfigMapName)
+	if err != nil {
+		log.Warningf("Unable to get certificate configmap [%s/%s]. Err: %s", controlPlane.IstiodNamespace, certificatesConfigMapName, err)
+	} else {
+		cert, _ := parseIstioControlPlaneCertificate(certConfigMap)
```

Comment:
The potential error is set on a "error" field, which is then show in the UI if it happens (nicely, unable to decode or parse). I will remove the returned error, because is not necessary.